src/api/fuel.ts

```diff
@@ -0,0 +1,559 @@
+/* eslint-disable quotes */
+/* eslint-disable prettier/prettier */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
I would prefer we discuss the linting rules and change the rules as we deem appropriate than disabling rules on a file.

Reply:
we can add a grep task for this to the current linting task to not hinder lingering PRs